docs/node/run-your-node/paratime-observer-node.mdx

```diff
@@ -0,0 +1,143 @@
+
+### Stake Requirements
+
+To be able to register as a ParaTime observer node on the Oasis Network, you need to have enough tokens staked in your
+entity's escrow account.
+
+Current minimum staking requirements for a specific ParaTime are listed on the [Stake Requirements] page. Should you
+want to check the staking requirements for other node roles and registered ParaTimes manually, use the Oasis CLI tools
+as described in [Common Staking Info].
+
+Finally, to stake the tokens, use our [Oasis CLI tools]. If everything was set up correctly, you should see something
+like below when running [`oasis account show`] command for your entity's account (this is an example for Testnet):
+
+![code shell](../../../external/cli/examples/account/show-delegations.in.static)
+
+![code](../../../external/cli/examples/account/show-delegations.out.static)
+
+:::caution
+
+The stake requirements may differ from ParaTime to ParaTime and are subject to change in the future.
+
+:::
+
+:::info
+
+For example, if want to register an observer node for Testnet/Mainnet currently need to self-delegate **200 TEST/ROSE**
+tokens:
+- **100 TEST/ROSE** for registering a new node entity and,
+- **100 TEST/ROSE** for gaining an observer role.
+
+:::
+
+[Stake Requirements]: prerequisites/stake-requirements.md
+[Run a ParaTime Node]: ../../get-involved/run-node/paratime-node.mdx
+[Common Staking Info]: ../../general/manage-tokens/cli/network.md#show-native-token
+[Oasis CLI tools]: ../../general/manage-tokens/cli/account.md#delegate
+[`oasis account show`]: ../../general/manage-tokens/cli/account.md#show
+
+### Register a New Entity or Update Your Entity Registration
+
+:::danger
+
+Everything in this section should be done on the `localhost` as there are sensitive items that will be created.
+
+:::
+
+:::danger
+
+If you plan to run an observer node for Mainnet and Testnet make sure you create and use two separate entities
+to prevent replay attacks.
+
+:::
+
+1. If you don't have an entity yet, create a new one by following the [Initialize Entity] instructions for validators.
+
+2. If you will be running the ParaTime on a new Oasis node, also initialize a new node by following the
+   [Starting the Oasis Node] instructions for validators.
+
+3. Now, [list your node ID] in the entity descriptor file `nodes` field.
+
+4. [Register] the updated entity descriptor.
+
+5. By adding the created entity ID in the node config file, you will [configure the node] to automatically register for
+   the roles it has enabled (i.e. observer role) via the `registration.entity_id` configuration flag. No manual node
+   registration is necessary.
+
+   ```yaml
+   # ... sections not relevant are omitted ...
+   registration:
+     entity_id: {{ entity_id }}
+   ```
+
+6. Once the registration is complete, please share the Entity IDs with us so that we can whitelist them accordingly.
+
+:::tip
+
+In the future, the whitelisting process may no longer be required.
+
+:::
```

Comment:
This actually depends on the paratime so I would omit this or mention that there may be per-paratime on-chain policy requirements for running observer nodes.

Reply:
Got it, updated the tip to mention that there may be per-runtime on-chain policy requirements.